Modules/_ctypes/_ctypes.c

```diff
@@ -5476,18 +5476,24 @@ comerror_init(PyObject *self, PyObject *args, PyObject *kwds)
     return 0;
 }
 
+static int
+comerror_clear(PyObject *self) {
```

Comment:
All new code should follow PEP-7.

```suggestion
comerror_clear(PyObject *self)
{
```